search_events_app/services/db/db_response_processor.py

```diff
@@ -10,28 +10,35 @@ def process_events(data):
                 'name': item[1],
                 'category': item[2],
                 'format_': item[3],
-                'organizer': item[4],
+                'organizer': get_organizer(item),
                 'country': get_country(item),
-                'start_date': item[6],
+                'start_date': item[7],
                 'language': get_language(item),
-
             }
             events.append(event_dict)
     return events
 
 
 def get_country(item):
     try:
-        country = Country.objects.get(alpha_2_code=item[5])
+        country = Country.objects.get(alpha_2_code=item[6])
         return country.name
     except Exception:
         return None
 
 
 def get_language(item):
-    lang = item[7]
+    lang = item[8]
     return lang.split('_')[0]
 
 
 def get_url(item):
     return f'https://www.eventbrite.com/e/{item[0]}'
+
+
+def get_organizer(item):
+    if item[4]:
+        return item[4]
+    if item[5]:
+        return item[5]
+    return None
```

Comment:
I don't understand this behaviour... could you explain to me? There is another way to access these values?

Reply:
The QuerySet is like a list of tuples, so you have to access each value of that tuple via the index (e.g. item[0]). The order of each column is the same one that we request in the SELECT to the DB